packages/session-replay-browser/src/config/joined-config.ts

```diff
@@ -89,16 +88,10 @@ export class SessionReplayJoinedConfigGenerator {
     }
 
     // Override all keys in the local config with the remote privacy config.
-    if (privacyConfig && Object.keys(privacyConfig).length > 0) {
-      if (!config.privacyConfig) {
-        config.privacyConfig = {};
-      }
-
-      for (const key in privacyConfig) {
-        const k = key as keyof PrivacyConfig;
-        config.privacyConfig[k] = privacyConfig[k];
-      }
-    }
+    config.privacyConfig = {
```

Comment:
if remote privacyConfig is empty, we should keep the privacyConfig set through the SDK for backwards compat.

Reply:
Yeah I thought that was happening here, did I miss something?

Reply:
no, i just don't know enough about js =).